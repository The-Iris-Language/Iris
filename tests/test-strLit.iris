.. test for string literals all cases: empty string, whitespace, normal str
class Main () { 
  public: 
    int main() {
      string s;
      s = "'a'";
      s = "";
      s = " ";
      return 0;
    }
}