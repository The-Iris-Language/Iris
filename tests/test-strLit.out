class Main() { 
int main() {
string s;
s = "'a'";
s = "";
s = " ";
return 0;
}
}